### DISCUSSION TITLE
Creation of Home brew cask to ease installation of rednotebook in mac os

### DISCUSSION THREAD

**Discussion by kishaningithub** [jendrikseipp/rednotebook#628] [Ideas]
As of now i see steps documented for installation using homebrew in mac os here

https://github.com/jendrikseipp/rednotebook/blob/master/dev/mac-homebrew.txt

It would be great if a formula/cask can be created so that users can just do `brew install rednotebook` and get it installed :-)

## Example

- Keepass XC
  - homebrew cask definition - https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/keepassxc.rb
  - Repo - https://github.com/keepassxreboot/keepassxc

**Comment by jendrikseipp** [comment]
I fully agree. Unfortunately, I have no macOS experience to create this. Any help is appreciated!
